x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/sequence/SequenceKey.java

```diff
@@ -15,14 +17,16 @@
 
 import static java.util.Collections.emptyList;
 
-public class SequenceKey {
+public class SequenceKey implements Accountable {
+
+    private static final long SHALLOW_SIZE = RamUsageEstimator.shallowSizeOfInstance(SequenceKey.class);
 
     public static final SequenceKey NONE = new SequenceKey();
 
     private final Object[] keys;
     private final int hashCode;
 
-    SequenceKey(Object... keys) {
+    public SequenceKey(Object... keys) {
```

Comment:
Why public?

Reply:
Moved the Tests into the `sequence` package to avoid this, so I revert it.